src/components/FilterTodos/FilterTodos.tsx

```diff
@@ -0,0 +1,89 @@
+import classNames from 'classnames';
+import React, { useEffect, useState } from 'react';
+import { GroupBy } from '../../types/GroupBy';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  filterTodos: (groupBy: string) => void;
+  completedTodosLength: number;
+  leftTodosLength: number;
+  todos: Todo[];
+  removeCompletedTodos: () => void;
+};
+
+export const FilterTodos: React.FC<Props> = ({
+  filterTodos,
+  todos,
+  completedTodosLength,
+  leftTodosLength,
+  removeCompletedTodos,
+}) => {
+  const [groupBy, setGroupBy] = useState('All');
+
+  useEffect(() => {
+    filterTodos(groupBy);
+  }, [groupBy, todos]);
+
+  const handleChange = (event: React.SyntheticEvent<HTMLAnchorElement>) => {
+    setGroupBy(event.currentTarget.textContent || 'All');
+  };
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${leftTodosLength} items left`}
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          data-cy="FilterLinkAll"
+          href="#/"
+          className={classNames(
+            'filter__link',
+            { selected: groupBy === GroupBy.All },
+          )}
+          onClick={handleChange}
+        >
+          All
+        </a>
+
+        <a
+          data-cy="FilterLinkActive"
+          href="#/active"
+          className={classNames(
+            'filter__link',
+            { selected: groupBy === GroupBy.Active },
+          )}
+          onClick={handleChange}
+        >
+          Active
+        </a>
+        <a
+          data-cy="FilterLinkCompleted"
+          href="#/completed"
+          className={classNames(
+            'filter__link',
+            { selected: groupBy === GroupBy.Completed },
+          )}
+          onClick={handleChange}
+        >
+          Completed
+        </a>
+      </nav>
+
+      {completedTodosLength ? (
+        <button
+          data-cy="ClearCompletedButton"
+          type="button"
+          className="todoapp__clear-completed"
+          onClick={removeCompletedTodos}
+          style={{
```

Comment:
use classnames library

Reply:
We already have class `hidden` which we use in `ErrorNotification`.
Here better implement the same way. Do not pass styles this way - it is a good practice(don't do this way in future test tasks, only if task description said it), better crate separate class/classes for this purpose.